Update community resources for migration guides (#11376)
withastro/docs: src/content/docs/en/guides/cms/drupal.mdx

### Publishing your site

To deploy your website, visit our [deployment guides](/en/guides/deploy/) and follow the instructions for your preferred hosting provider. 

## Community Resources

<CardGrid>

  <LinkCard title="Create a web application with Astro and Drupal" href="https://www.linkedin.com/pulse/create-web-application-astrojs-website-generator-using-gambino-kx6cf"/>

</CardGrid>

:::note[Have a resource to share?]

> If you found (or made!) a helpful video or blog post about using Drupal with Astro, edit this page and add it here!

Author: ```suggestion
If you found (or made!) a helpful video or blog post about using Drupal with Astro, [add it to this list](https://github.com/withastro/docs/edit/main/src/content/docs/en/guides/cms/drupal.mdx)!
```
